### PR TITLE
Fix/style text cleanup

### PR DESCRIPTION
**Pull request by EgoEris**
## Description

Adding in some more images, fixing up aesthetics and such

### CODE REVIEW

**Comment by fung-alvin** [comment]
Should we update the styling of our fonts and headings here in this branch? There are still inconsistencies, such as titles being in full caps vs Pascal Case, green coloured vs white, bold vs normal weight. Additionally, size of body text.

**Comment by fung-alvin** [comment]
Another thing is the width of our text on the screen. Do we want to limit it to 90% of the width of the screen?
I used this style in my divs:
`          <div
            style={{
              width: "90%",
              margin: "0 auto",
              textAlign: "center",
            }}
          >`

**Comment by EgoEris** [comment]
> Should we update the styling of our fonts and headings here in this branch? There are still inconsistencies, such as titles being in full caps vs Pascal Case, green coloured vs white, bold vs normal weight. Additionally, size of body text.

Ahh good catch! I'll take of that real quick

File: app/layout.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default function RootLayout({
         className={inter.className + " bg-black flex flex-col min-h-screen"}
       >
         <Navbar />
-        <div className="flex-1">{children}</div>
+        <div className="flex-1 " style={{width: "95%", margin: "0 auto", textAlign: "center",} } >{children}</div>
```
Comment (fung-alvin) [inline app/layout.tsx:25]: This limits full width images to 95% of screen as well. Do we want that?

**Review by fung-alvin — APPROVED** [review]

File: app/layout.tsx (hunk 1 above)
Comment (EgoEris) [reply app/layout.tsx:25]: Hmm true, I think for now I'll leave it be, however post deployment we can potentially figure out how to do this efficiently for the text sections

**Pull request merged**